v5_ws_private.go

```diff
@@ -212,7 +220,7 @@ func (s *V5WebsocketPrivateService) Run() error {
 
 // Ping :
 func (s *V5WebsocketPrivateService) Ping() error {
-	if err := s.connection.WriteMessage(websocket.PingMessage, nil); err != nil {
+	if err := s.connection.WriteMessage(websocket.TextMessage, []byte(`{"op":"ping"}`)); err != nil {
```

Comment:
https://bybit-exchange.github.io/docs/v5/ws/connect#how-to-send-the-heartbeat-packet
>// req_id is a customised ID, which is optional
ws.send(JSON.stringify({"req_id": "100001", "op": "ping"}));